xla/service/gpu/gpu_compiler.cc

```diff
@@ -1239,6 +1261,8 @@ absl::Status GpuCompiler::OptimizeHloModule(
       hlo_module, stream_exec, options, gpu_target_config,
       thread_pool.get_mutable()));
 
+  TF_RETURN_IF_ERROR(RunCollectivePipelinerPasses(hlo_module));
```

Comment:
It is better to rename and make it clear this is post layout CP: RunPostLayoutCollectivePipelinerPasses

Reply:
Done. PTAL.